test/buildsetup.sh

```diff
@@ -26,6 +26,12 @@ else
 	echo "*** Reusing test directory: $TARGETDIR"
 fi
 
+if [ "$SPACK_CCACHE" = "true" ]
+then
+  CCACHE_CACHEDIR="$(ccache -k cache_dir)"
+  CCACHE_MAXSIZE="$(ccache -k max_size)"
+fi
+
```

Comment:
This looks like you're transferring specific options. What if another option is being added later on?

What about symlinking the appropiate directories into the new HOME?

Reply:
> What about symlinking the appropiate directories into the new HOME?

I thought about symlinking the config file, but then I would have to write the cache dir explicitely into the config file and one would have to know to not remove it even though it is the default path. Hmm, I could copy the config file and write the cache dir explicitely..

Reply:
Well, I meant symlink the config file and symlink the cache directory (it's probably relative to the home?)

But anyway: If this works, merge it.

Reply:
ya, let's refine, when it breaks.